test-cases/features/recovery_10min.yaml

```diff
@@ -0,0 +1,20 @@
+# TODO: need to qualify
+test_duration: 40
+
+stress_cmd: "cassandra-stress write no-warmup cl=QUORUM duration=3m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=50 -pop seq=1..10000000 -log interval=5"
```

Comment:
instead of using "duration=" use n= 10000000" (or the amount of keys you would like to write).
Check how long it takes to write it, if more than 2-3 mins, reduce it.

Reply:
took 8 min for the write, will reduce it to 1,500,000